e2e/__tests__/customMatcherStackTrace.test.js

```diff
@@ -23,3 +23,11 @@ test('works with custom matchers', () => {
 
   expect(rest).toMatchSnapshot();
 });
+
+test('custom async matchers', () => {
+  const {stderr} = runJest('custom-matcher-stack-trace', ['async.test.js']);
+
+  const result = extractSummary(stderr);
+
+  expect(result).toMatchSnapshot();
```

Comment:
Another thing - I merged #7651, so can you rebase/merge master and add `wrap` around it here?

Reply:
Sure thing. Nice work on that. 🔥 I just added it to my commit.